community/modules/scripts/htcondor-install/files/install-htcondor-autoscaler-deps.yml

```diff
@@ -24,13 +24,13 @@
   - name: Create virtual environment for HTCondor autoscaler
     ansible.builtin.pip:
       name: pip
-      version: 21.3.1  # last Python 2.7-compatible release
+      version: 21.3.1  # last Python 3.6-compatible release
       virtualenv: /usr/local/htcondor
       virtualenv_command: /usr/bin/python3 -m venv
   - name: Install latest setuptools
     ansible.builtin.pip:
       name: setuptools
-      state: 44.1.1  # last Python 2.7-compatible release
+      version: 59.6.0  # last Python 3.6-compatible release
```

Comment:
Reminder to run integration tests for merge to main.

Reply:
I thought these automatically ran. I will trigger them once I address the other issues.

Reply:
no need since not a hot fix to main.